pkg/config/draftconfig.go

```diff
@@ -24,13 +24,13 @@ type BuilderVar struct {
 	Description      string   `yaml:"description"`
 	VarType          string   `yaml:"type"`
 	ExampleValues    []string `yaml:"exampleValues"`
-	IsPromptDisabled bool     `yaml:"disablePrompt"`
```

Comment:
what does moving the disabled prompting feature from the BuilderVar to the BuilderVarDefault accomplish? currently there can be multiple builderVarDefaults for a single variable Name, so in this case it would disable if any of the defaults for a var Name are disabled.